### PR TITLE
Move numerical test of BatchBoxCox operator to OperatorTest

### PR DESCRIPTION
**Pull request by ssmetkar**
*Description*: Move numerical test of BatchBoxCox operator to OperatorTest
*Testing*: Running unit test cases. All test cases passed.
*Documentation*: No documentation updated needed.
[Optional Fixes #issue]

Please see a detailed explanation of how to fill out the fields in the relevant sections in PULL_REQUEST.md.

### CODE REVIEW

File: tests/unittests/Caffe2ImporterTest.cpp [hunk 1]
```diff
@@ -1115,41 +1115,11 @@ TEST(caffe2, batchBoxCox) {
   Tensor data(ElemKind::FloatTy, {kRows, kCols});
   Tensor lambda1(ElemKind::FloatTy, {kCols});
   Tensor lambda2(ElemKind::FloatTy, {kCols});
-  auto dataH = data.getHandle();
-  auto lambda1H = lambda1.getHandle();
-  auto lambda2H = lambda2.getHandle();
-
-  // Fill inputs with random values.
```
Comment (SplitInfinity) [inline tests/unittests/Caffe2ImporterTest.cpp]: Please add this comment back to the new location of the test.

File: tests/unittests/Caffe2ImporterTest.cpp [hunk 2]
```diff
@@ -1115,41 +1115,11 @@ TEST(caffe2, batchBoxCox) {
...
-  dataH.randomize(0.0, 5.0, mod.getPRNG());
-  lambda1H.randomize(1.0, 2.0, mod.getPRNG());
-  lambda2H.randomize(1.0, 2.0, mod.getPRNG());
-
-  // Zero out every other element to lambda1 to test that case of the transform.
```
Comment (SplitInfinity) [inline tests/unittests/Caffe2ImporterTest.cpp]: Please add this comment back to the new location of the test.

File: tests/unittests/Caffe2ImporterTest.cpp [hunk 3]
```diff
@@ -1115,41 +1115,11 @@ TEST(caffe2, batchBoxCox) {
...
   Tensor O(ElemKind::FloatTy, {kRows, kCols});
   auto OH = O.getHandle();
 
-  for (size_t i = 0; i < kRows; ++i) {
-    for (size_t j = 0; j < kCols; ++j) {
-      float d = dataH.at({i, j});
-      float l1 = lambda1H.at({j});
-      float l2 = lambda2H.at({j});
-
-      // Compute elementwise Box-Cox transform.
```
Comment (SplitInfinity) [inline tests/unittests/Caffe2ImporterTest.cpp]: Please add this comment back to the new location of the test.

File: tests/unittests/Caffe2ImporterTest.cpp [hunk 4]
```diff
@@ -1115,41 +1115,11 @@ TEST(caffe2, batchBoxCox) {
...
-      float tmp = std::max(d + l2, 1e-6f);
-      if (l1 == 0) {
-        // Clip argument to log and pow at 1e-6 to avoid saturation.
```
Comment (SplitInfinity) [inline tests/unittests/Caffe2ImporterTest.cpp]: Please add this comment back to the new location of the test.

**Comment by rdzhabarov** [comment]
@ssmetkar do you plan to land this?

**Pull request closed without merging**